### DISCUSSION TITLE
Is there any way to use UTC?

### DISCUSSION THREAD

**Discussion by VitorCioletti** [atifaziz/NCrontab#95] [Q&A]
Does this library support any cron expression to use UTC timezone?

**Comment by atifaziz** [comment]
Schedules are calculated irrespective of time zone so you it's up to the caller to decide how to have the resulting time interpreted and adjusted.

**Comment by rshillington** [comment]
@VitorCioletti here's a [dotnet fiddle script](https://dotnetfiddle.net/C80NhB) that may help in switching between local times and UTC. 

In our case, we store the NextOccurance in our database in UTC and we query in UTC, looking for jobs that are due to run.  However, we allow the user to specify the cron expression in their local time (which means we need their timezone)

In this script, I assume "America/Toronto" as the timezone, and get a schedule of runs between 9 and 17 local time.

Hope that helps.

**Comment by LordMike** [comment]
In theory, just "pretending" a DateTime has the proper timezone should do it. It won't handle anything like daylight saving times and so on though..

**Comment by FLAMESpl** [comment]
Actually UTC is the only zone this library can handle due to mentioned DST nuances.
